bench/fixture.ts

```diff
@@ -323,6 +325,22 @@ export class BenchFixture {
       }
     }
 
+    if (options.gasToken > 0) {
+      debug(`encoding ${options.gasToken} gasToken burns`);
+
+      // Create more gas tokens than needed as otherwise we might get extra storage refunds which may skew benchmarks
```

Comment:
I think there might be a similar issue in the first burning operation, at least for the CHI-based token: the first time that the token is burned the `totalBurned` slot is set from zero to nonzero, making the amount spent a bit imprecise on the first use. You could just create and burn tokens at the moment of first deployment of the contract.

Reply:
Nice catch!